frontend/src/lib/api/coReview.ts

```diff
@@ -0,0 +1,47 @@
+import { User } from "./auth";
+import { DestructionList } from "./destructionLists";
+import { request } from "./request";
+
+export type CoReview = {
+  destructionList: string;
+  listFeedback: string;
+  pk?: number;
+  author?: User;
```

Comment:
I am not sure what the convention is, but should there be a type for the "read" and for the "write" to make it clear that POSTing the author field will be ignored?

Reply:
Not sure, tbh I don't like any of the options :P 

Lets go with correctness, I've changed into a CoReviewBase type for the create a CoReview that extends it with additional fields created by the backend